Cartfile

```diff
@@ -1,2 +1,4 @@
-github "mapbox/mapbox-vision-ios" == 0.2.0
-github "mapbox/mapbox-navigation-ios" ~> 0.26.0
+github "mapbox/mapbox-vision-ios" "kobe/minor-bump-vision-sdk-0.3.0"
```

Comment:
I made it this a `minor` bump because the dependent SDK, `"MapboxCoreNavigation"` has gone through a series of minor releases `~>0.26.0` - `~> 0.29.0` since its last update. This needs to be a `0.3.0`.

Reply:
See discussion here https://github.com/mapbox/mapbox-vision-ios/pull/54#discussion_r258116704